api/cloud/v1.md

```diff
@@ -74,9 +74,29 @@ docs_area: reference.api
   function changeVersion() {
     const doc = document.getElementById('rd');
     selectedVersion = document.getElementById("versionSelector").value;
-    doc.loadSpec('https://cockroachlabs.cloud/assets/docs/ccapi_spec_' + selectedVersion + '.json');
+    doc.loadSpec('https://cockroachlabs.cloud/assets/docs/' + selectedVersion + '.json');
   }
 
+  window.addEventListener('load', (event) => {
+    const params = new Proxy(new URLSearchParams(window.location.search), {
```

Comment:
I had never seen this usage of Proxy before. Its kind of cool.  In this case however, I'd recommend just using the 3rd  option seen here: https://fedingo.com/how-to-get-query-string-values-in-javascript/. Since we're only getting a single parameter here we don't save much by changing the api.  It could just be:

```js
const urlParams = new URLSearchParams(window.location.search);
const version = urlParams.get('version');
```